api/src/plugins/email-token-scheme.js

```diff
@@ -1,10 +1,12 @@
 const Boom = require('boom');
-const bookshelf = require('../lib/bookshelf');
 
 exports.register = function (server, pluginOptions, next) {
   server.auth.scheme('email_token', (server, schemeOptions) => {
     return {
       async authenticate(request, reply) {
+        if (!request.query.token) {
+          return reply(Boom.forbidden());
+        }
```

Comment:
Without this, the `findToken()` call below throws when `request.query.token` is missing, resulting in a 500 error instead of 403.

Reply:
I just realized that this could be handled per-route via validation. Then, after a second thought, I realized that a 400 error might not be appropriate for a missing token, and this scheme should probably not depend on the route being configured correctly to avoid throwing a runtime error. Opinions?

Reply:
I like what you have here